plugins/harness-iacm/src/plugin.ts

```diff
@@ -0,0 +1,21 @@
+import {
+  createPlugin,
+  createRoutableExtension,
+} from '@backstage/core-plugin-api';
+
+import { rootRouteRef } from './routes';
+
+export const harnessIacmPlugin = createPlugin({
+  id: 'harness-iacm',
+  routes: {
+    root: rootRouteRef,
+  },
+});
+
+export const HarnessIacmPage = harnessIacmPlugin.provide(
```

Comment:
Please follow this Naming Pattern
https://backstage.io/docs/plugins/composability/#naming-patterns